c_utils.py

```diff
@@ -1,12 +1,6 @@
-import collections
-import glob
 import logging
 import os
 import pprint
-import re
-import sys
-
-import yaml
 
 # from shared_utils import overlaps, overlap_allowed, extension_overlap_allowed, instruction_overlap_allowed, process_enc_line, same_base_isa, add_segmented_vls_insn, expand_nf_field
```

Comment:
@Timmmm there are many related functions from `shared` module as frequently used across various scripts, listing them all will lead to reducing the readability of the code. The intention to use Wildcard imports was to declutter the import section, shifting the reader’s focus to the core logic rather than the long list of helper utilities. 

Moreover, it helps to keep the code more cleaner and concise, making the code easier to scan at a glance without a long list of imports at the top.
Hence, I guess these could be removed from commented out line.

Reply:
Best practice is to avoid wildcard imports in general. The conciseness is not worth the downsides. See

* http://docs.quantifiedcode.com/python-anti-patterns/maintainability/from_module_import_all_used.html
* https://pybit.es/articles/why-you-should-avoid-import-in-python/
* https://stackoverflow.com/questions/3615125/should-wildcard-import-be-avoided
* https://www.reddit.com/r/learnpython/comments/8zej2c/eli5_what_are_wildcard_imports_and_why_should_i/

However I don't think we need to worry about it in this PR. Let's keep this PR small.